osh/client/completion/_osh-cli

```diff
@@ -76,6 +76,8 @@ _osh-cli_args()
                 '(-r --regex)'{-r,--regex}'[query by regular expression (python, module: re)]'
                 '(-p --pacakge)'{-p,--package}'[query by package name]'
                 '(-n --nvr)'{-n,--nvr}'[query by NVR (default one)]'
+                '(-s --state)'{-s,--state}'[query by task state. This option is used \
```

Comment:
Sorry, I overlooked this in the previous review round.  The `-s` option takes an argument so the completions should reflect that as well.  I'll post a patch, hopefully this week, that also implements the automated completions of task states themselves so that you don't have to learn the magic behind `zsh` and `bash` completions.